apps/src/code-studio/pd/application_dashboard/quick_view_table.jsx

```diff
@@ -143,93 +138,61 @@ export class QuickViewTable extends React.Component {
       }
     );
 
-    if (this.showLocked()) {
-      columns.push({
```

Comment:
Thanks for tracking all the little ways 'facilitator' is worked into all this logic! Seems like a lot of moving parts!